### ISSUE TITLE
Combine template library repositories

### ISSUE THREAD

**Issue by jrha** [quattor/release#309]
Merge template-library-core and template-library-standard into a single repository named template-library, the history must be preserved https://github.com/unravelin/tomono may help with this.

**Comment by jrha** [comment]
* Documentation will also need to be updated to reflect new locations.
* Release and build_all_repos scripts will need to ne updated to operate on new locations.

**Comment by ned21** [comment]
Agreed at RAL workshop that it would be nice to consolidate all templates into one repository but first need agreement on whether we can move away from the per-release development workflow which would cause a single repo to have an unmaintainable number of branches.

**Comment by jrha** [comment]
Individual repository indentity should translate to subdirectories, e.g.
Files from `template-library-core` should go in to a subdirectory named `core`.

**Comment by jrha** [comment]
Not likely to happen without actual time put aside to do it, which I can't promise right now.

**Comment by jouvin** [comment]
I completely forgot about this issue... I'm personnally not in favor of merging these repos with different purposes... But restructuring the repositories not using a master branch is something we should do and that should not be too difficult.
